probe/endpoint/ebpf.go

```diff
@@ -0,0 +1,256 @@
+package endpoint
```

Comment:
(minor) Ideally, this should only be compiled in linux. Not that we care too much about macs these days.